easybuild/easyconfigs/m/Mono/Mono-4.6.2.7.eb

```diff
@@ -0,0 +1,17 @@
+easyblock = 'ConfigureMake'
```

Comment:
it's great that they cleaned things up so you can now install Mono using just `configure`-`make`-`make install`

but since we *have* an easyblock for Mono, shouldn't we update it so it falls back to just `ConfigureMake` if the Mono version is recent enough?